### PR TITLE
Add TokenLifetimePolicy data.

### PR DESCRIPTION
**Pull request by georgend**
Add TokenLifetimePolicy data.

### CODE REVIEW

File: scripts/TenantSetup/TenantHttpData/Policy/Policy.ps1 [hunk 1]
```diff
@@ -52,10 +52,21 @@ if($null -eq $currentTokenIssuancePolicy){
     $currentTokenIssuancePolicy = Invoke-RequestHelper -Uri "policies/tokenIssuancePolicies" -Method POST -Body $tokenIssuancePolicyData
     $currentTokenIssuancePolicy.id
 }
+#Create tokenLifetimePolicy https://docs.microsoft.com/en-us/graph/api/tokenlifetimepolicy-post-tokenlifetimepolicies?view=graph-rest-1.0&tabs=http
+$tokenLifetimePolicyData = Get-RequestData -ChildEntity "TokenLifetimePolicy"
+$currentTokenLifetimePolicy = Invoke-RequestHelper -Uri "policies/tokenLifetimePolicies" -Method GET |
```
Comment (baywet) [inline scripts/TenantSetup/TenantHttpData/Policy/Policy.ps1:57]: use a variable for the URL to avoid duplication

**Review by baywet — CHANGES_REQUESTED** [review]

File: scripts/TenantSetup/TenantHttpData/Policy/Policy.ps1 (hunk 1 above)
Comment (georgend) [reply scripts/TenantSetup/TenantHttpData/Policy/Policy.ps1:57]: Please elaborate. Which URL is duplicated?

Comment (baywet) [reply scripts/TenantSetup/TenantHttpData/Policy/Policy.ps1:57]: `policies/tokenLifetimePolicies` here and line 62

Comment (georgend) [reply scripts/TenantSetup/TenantHttpData/Policy/Policy.ps1:57]: Yes, I just realized it. Thanks.

**Review by baywet — APPROVED** [review]
thanks for taking in the changes

**Pull request merged**